app/assets/javascripts/components/option-select.js

```diff
@@ -238,9 +266,27 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     }
 
```

Comment:
Add a note - there are no tests for the rest of this method.  All tests `return` before.